docs/source/getting-started/quickstart-2/index.rst

```diff
@@ -91,10 +91,12 @@ parameters, results, and model itself of each trial as a child run.
 
     def train_model(params, epochs, train_x, train_y, valid_x, valid_y, test_x, test_y):
         # Define model architecture
+        mean = np.mean(train_x, axis=0)
+        var = np.var(train_x, axis=0)
```

Comment:
why can't it use global mean and var ?

Reply:
oh sorry,
i meant compute `mean` of all elements in `train_x` array. Similar to `var`, like
`mean = np.mean(train_x)`

Reply:
It will give only one value as a result, but we need an array with a shape of [1,11].